CMakeLists.txt

```diff
@@ -54,7 +62,7 @@ if(OS_WINDOWS)
                  ${CMAKE_BINARY_DIR}/installer-Windows.generated.iss)
 
   if(MSVC)
-    target_compile_options(${CMAKE_PROJECT_NAME} PRIVATE /MP /d2FH4-)
+    target_compile_options(${CMAKE_PROJECT_NAME} PRIVATE /W4)
```

Comment:
Why not putting it in the helper file ?

Reply:
Mainly because I want developers to actively remove this and to have an "example" of a compiler option in the template code (to avoid them using `add_options` instead).

Reply:
Ok, can be marked as resolved then.